micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/ProductPage/ProductPageComponent.js

```
@@ -0,0 +1,63 @@
+      <div className="products-list">
+        {ProductsConfig?.products.map((product, index) => (
+          <Card key={index} className="product-card">
+            <div className="product-header">
+              {Digit.Utils.iconRender(product.icon,"#c84c0e")}
+              <h2 className="product-title">{t(product.title)}</h2>
+            </div>
+            <p className="product-description">{t(product.description)}</p>
+            <Button
+              className="explore-button"
+              variation="secondary"
+              label={t("COMMON_EXPLORE") + " →"}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)